src/tests/harness/BenchmarkDotNet.Extensions.Tests/BenchmarkDotNet.Extensions.Tests.csproj

```diff
@@ -1,9 +1,10 @@
-﻿<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <Import Project="$(MSBuildThisFileDirectory)\..\..\..\common.props" />
   <PropertyGroup>
     <IsPackable>false</IsPackable>
+    <TargetFrameworks>netcoreapp3.0</TargetFrameworks>
```

Comment:
Have you ever noticed that VS was never showing the source code files for this project and it was also not recognized as test project?

![obraz](https://user-images.githubusercontent.com/6011991/60245416-dbc67200-98bc-11e9-9dec-7a92be224015.png)

Setting TFMs to single TFM fixes that. I have no idea why, but it's enough to have just one.

The test project is now recognized as test project and VS shows the tests in Test Explorer

![obraz](https://user-images.githubusercontent.com/6011991/60245725-76bf4c00-98bd-11e9-84ce-2c61cae6baca.png)